src/features/scheduled-messages.ts

```diff
@@ -1,85 +1,152 @@
+
+How to ask for programming help <http://wp.me/p2oIwo-26>
+How do I ask a good question <https://stackoverflow.com/help/how-to-ask>
+`,
+    },
+  },
+  {
+    postTo: [{ interval: FREQUENCY.often, channelId: CHANNELS.helpReact }],
+    message: {
+      content: `This channel is good for specific questions about React, how React's features work, or debugging a small (< ~50 lines of code) snippet of JS that uses React. Question not getting answered? Maybe it's hard to answer, check out these resources for how to ask a good question:
+
+How to ask for programming help <http://wp.me/p2oIwo-26>
+How do I ask a good question <https://stackoverflow.com/help/how-to-ask>
+`,
+    },
+  },
+  {
+    postTo: [
+      { interval: FREQUENCY.moreThanWeekly, channelId: CHANNELS.helpReact },
+    ],
+    message: {
+      content: `Check our the other channels too! This is our highest-traffic channel, which may mean your question gets missed as other discussions happen.
+
+#help-js For questions about pure Javscript problems.
+#help-styling For questions about CSS or other visual problems.
+#help-backend For questions about issues with your server code.
+#code-review Get deeper review of a snippet of code.
+#jobs-advice If you have a question about your job or career, ask it in here.
+#general-tech Discussion of non-JS code, or that new laptop you're deciding on.
+#tooling for questions about building, linting, generating, or otherwise processing your code.
+
+Looking for work? Trying to hire? Check out #job-board, or <https://reactiflux.com/jobs>
+
+Has someone been really helpful? Shoutout who and what in #thanks! We keep an eye in there as one way to find new MVPs. Give us all the reactions in there too!
+
+Please remember our Code of Conduct: <https://reactiflux.com/conduct>
+and our guidelines for promotion: <https://reactiflux.com/promotion>
+
+If you see anything that violates our rules, help alert the mods by reacting to it with 👎
+`,
+    },
+  },
+  {
+    postTo: [
+      { interval: FREQUENCY.moreThanWeekly, channelId: CHANNELS.random },
+    ],
+    message: {
+      content: `Have you read our Code of Conduct? <https://reactiflux.com/conduct> Don't make dad angry.
+
+If something crosses a line, give it a 👎, or if you'd prefer to remain anonymous, let mods know with the form at <https://reactiflux.com/contact>`,
+    },
+  },
+];
 
 export const messages: MessageConfig[] = [];
 
-const isTextChannel = (
-  channel: discord.Channel,
-): channel is discord.TextChannel | discord.DMChannel | discord.NewsChannel => {
-  return (
-    channel.type === "GUILD_TEXT" ||
-    channel.type === "GUILD_NEWS" ||
-    channel.type === "GUILD_STORE"
-  );
+export const scheduleMessages = (bot: discord.Client) => {
+  bot.on("ready", () => {
+    MESSAGE_SCHEDULE.forEach((messageConfig) =>
+      sendMessage(bot, messageConfig),
```

Comment:
Turns out the bug was that I was trying to send messages before the bot logged in, not anything related to permissions/etc 🤦 oh well, we got there